reflection_refraction.py

```diff
@@ -20,13 +20,14 @@ def check_if_posible(alfa, n1, n2):
         exit()
 
 def get_vars():
-    alfa = float(input("\nEnter the degrees of alfa: "))
+    alfa = (round(float(input("\nEnter the degrees of alfa: ")), 4))
```

Comment:
Why do you need this rounding? Is it only to have nicer prints?

Reply:
Float is calculating too much numbers after the comma and I think the equations turn out to be wrong.

Reply:
The float can't be a problem for the computation. 

Most probably the equation crashes because you are working out of the valid range for your trigonometric function.